App/Models/TextileTypes.ts

```diff
@@ -101,9 +101,7 @@ export type SharedImage = {
   path?: string,
   height: number,
   width: number,
-  isVertical: boolean,
-  caption?: string, // used for pre-upload display
```

Comment:
Don't need `caption` and `addResult` in here because we track them next to `SharedImage` in `ProcessingImagesRedux` now

Reply:
nice